src/SSHDebugPS/UI/ViewModels/ContainerPickerViewModel.cs

```diff
@@ -162,6 +165,41 @@ private void RefreshContainersListInternal()
                     containers = DockerHelper.GetRemoteDockerContainers(connection, Hostname, out totalContainers);
                 }
 
+                string serverOS;
+                bool getServerOS = DockerHelper.TryGetServerOS(Hostname, out serverOS);
+
+                if (getServerOS)
```

Comment:
remove `getServerOs` and change to `if (DockerHelper.TryGetServerOS(Hostname, out serverOS))`